src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -755,6 +755,37 @@ GUI_TEST_CLASS_DEFINITION(test_7384_2) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7401) {
+    // 1. Open human_T1.fa.
+    GTFileDialog::openFile(os, dataDir + "samples/FASTA/human_T1.fa");
+
```

Comment:
wait until task is finished/view is opened before using view related API. View-releated API will not necessary wait.

Reply:
Don't get the idea. As far as I know, `openFile` has `waitForMainThread` inside

Reply:
waitForMainThread is a wait for QT event loop, here we need to wait until document load/view-open task is finished.
You need to add
`GTUtilsTaskTreeView::waitTaskFinished(os);`
or at least
`checkSequenceViewWindowIsActive() `
before calling helpers that work over the existing sequence view.

Reply:
OK